### DISCUSSION TITLE
blog/se-l-america-avesse-il-bidet

### DISCUSSION THREAD

**Discussion by giscus** [peppocola/pompelmo#10] [Q&A]
# blog/se-l-america-avesse-il-bidet

Vivendo il sogno americano in attesa del badge per la NASA.

https://www.pompelmo.xyz/blog/se-l-america-avesse-il-bidet

**Comment by micdipper** [comment]
Like a questo commento per fargli fare il tour della casa (decido io della tua vita)

**Comment by micdipper** [comment]
Tutto giusto, a parte death note sopravvalutato!!!!

**Reply by peppocola** [reply]
sec me un po' noiosetto :P
